WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -127,30 +130,23 @@ class NotificationsViewController: UITableViewController {
         tableViewHandler.updateRowAnimation = .none
     }
 
-    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
-        super.viewWillTransition(to: size, with: coordinator)
-        // Note: We're assuming `tableViewHandler` might be nil. Weird case in which the view
-        // hasn't loaded, yet, but the method is still executed.
-        tableViewHandler?.clearCachedRowHeights()
-    }
-
 
     // MARK: - UITableView Methods
 
-    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
-        return nil
+    override func tableView(_ tableView: UITableView, estimatedHeightForHeaderInSection section: Int) -> CGFloat {
```

Comment:
So for these, since we end up returning a cached value, potentially, I myself felt it cleaner to keep both related methods next to each other, for code clarity.

But I'm going to refactor to reuse all the caching code across the app, anyways. So if you're OK with leaving it for now I can clean it up a bit later.